src/app/lib/layoutGrid.js

```diff
@@ -45,6 +45,11 @@ const fiveOfSixColumnsMaxWidthScaleable = `83.33%`;
 const fourOfSixColumnsMaxWidthScaleable = `66.67%`;
 // (4 / 6) * 100 = 66.6666.. = 66.67%
 
+const nestedGrid = css`
+  display: grid;
+  grid-column-gap: 16px;
```

Comment:
You should use `GEL_SPACING_DBL` instead of `16px` here. It's defined in `@bbc/gel-foundations/spacings`